metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -36,7 +36,7 @@ class Messages(icons: Icons) {
       new MessageActionItem("Import changes")
     def params: ShowMessageRequestParams = {
       val params = new ShowMessageRequestParams()
-      params.setMessage("sbt build needs to be re-imported")
```

Comment:
It's used in places where we don't have info about workspace - this could be possible, but I think it's more work than it's worth

Reply:
What about passing in the build tool name as a string? I think it would be helpful to know from the message what kind of build was detected,some workspaces might have both a gradle and sbt build (although I suspect it's uncommon).

Reply:
Changed to string and reworked the logic behind the fake LSP client - I only check for what the message contains.